src/databricks/labs/remorph/reconcile/query_builder/threshold_query.py

```diff
@@ -29,3 +50,69 @@ def build_query(self) -> str:
     def _construct_threshold_query(table, query_filter, col_expr) -> str:
         expr = ",".join(col_expr)
         return f"select {expr} from {table} where {query_filter}"
+
+    # Comparison query
+    def build_comparison_query(self) -> str:
+        select_clause, where = self._generate_select_where_clause()
+        from_clause, join_clause = self._generate_from_and_join_clause()
+        return select(*select_clause).from_(from_clause).join(join_clause).where(where).sql(dialect=Databricks)
+
+    def _generate_select_where_clause(self) -> tuple[list[exp.Expression], exp.Expression]:
```

Comment:
```suggestion
    def _generate_select_where_clause(self) -> tuple[list[exp.Select], exp.Where]:
```

isn't this is the right return type?

Reply:
It should be like `tuple[list[Alias], Or]` Changed it